CMakeLists.txt

```diff
@@ -33,6 +33,32 @@ endif( Psyllid_ENABLE_ITERATOR_TIMING )
 # dependencies #
 ################
 
+# FFTW
+option( Psyllid_ENABLE_FFTW "Flag to enable FFTW features" TRUE )
+if (Psyllid_ENABLE_FFTW)
+    find_package(FFTW REQUIRED)
+else (Psyllid_ENABLE_FFTW)
+    set (FFTW_FOUND FALSE)
+endif (Psyllid_ENABLE_FFTW)
+if (FFTW_FOUND)
+    add_definitions(-DFFTW_FOUND)
+    pbuilder_add_ext_libraries (${FFTW_LIBRARIES})
+    if (FFTW_THREADS_FOUND)
+        set (FFTW_NTHREADS 1 CACHE STRING "Number of threads to use for FFTW processes")
+        add_definitions (-DFFTW_NTHREADS=${FFTW_NTHREADS})
+        message (STATUS "FFTW configured to use up to ${FFTW_NTHREADS} threads.")
+    else (FFTW_THREADS_FOUND)
+        remove_definitions (-DFFTW_NTHREADS=${FFTW_NTHREADS})
+    endif (FFTW_THREADS_FOUND)
+else (FFTW_FOUND)
+    message(STATUS "Building without FFTW")
+    remove_definitions(-DFFTW_FOUND)
+    remove_definitions (-DFFTW_NTHREADS=${FFTW_NTHREADS})
+    # do I really want to do this?... not found so why set the include?
+    #set (FFTW_INCLUDE_DIR ${PROJECT_SOURCE_DIR}/External/FFTW)
```

Comment:
This is present in Katydid's CMakeLists.txt file because if FFTW is not present, I include a typedef for `fftw_complex` so that that type can be used regardless.